packages/react-instantsearch-core-next/src/useIndex.ts

```diff
@@ -0,0 +1,23 @@
+import index from 'instantsearch.js/es/widgets/index/index';
+import { useEffect, useMemo } from 'react';
+
+import { useIndexContext } from './useIndexContext';
+
+import type { IndexWidgetParams } from 'instantsearch.js/es/widgets/index/index';
+
+export type UseIndexProps = IndexWidgetParams;
+
+export function useIndex(props: UseIndexProps) {
+  const searchIndex = useIndexContext();
```

Comment:
```suggestion
  const parentIndex = useIndexContext();
```

this makes more sense to me

Reply:
That's confusing to me since you're in `useIndex`, and the "parent index" here makes me think that's the parent index of this index.

Reply:
isn't it the parent of the index?

Reply:
Ugh, you're right!

Reply:
See 0b0b82bfa339ec35f389d539841b6f6e7c509130.